benefit/data/sheets/CaseField/CaseField-WA.json

```diff
@@ -6,4 +6,4 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "assignedCaseRoles", "Label": "Assigned Case Roles", "FieldType": "DynamicList", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "previouslyAssignedCaseRoles", "Label": "Assigned Case Roles", "FieldType": "DynamicList", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "24/08/2023", "CaseTypeID": "Benefit", "ID": "issueInterlocDecisionDate", "Label": "Issue interloc decision date", "FieldType": "Date", "SecurityClassification": "PUBLIC"}
-]
```

Comment:
could we get rid of these whitespace changes so that it doesn't confuse things

Reply:
done